### DISCUSSION TITLE
Minimum brightness is below the sun-display-gui.

### DISCUSSION THREAD

**Discussion by ghost** [MonitorControl/MonitorControl#1313] [General]
### Before opening the issue, have you...?

- [X] Searched for existing issues
- [X] Looked through [the wiki](https://github.com/MonitorControl/MonitorControl/wiki)
- [X] Updated MonitorControl to the latest version (if applicable)

### Describe the bug

I have a Samsung CRG9. When I set the brightness to minimum, the bar display (☀️) is below the lower limit.

### Steps to reproduce

Adjust to minimum and display shows minimum.

### Expected behavior

The brightness should be change linear with the display.

### Environment Information (please complete the following information)

```markdown
- macOS version: 13.2
- Mac model: MacBook Air M1
- MonitorControl version: 4.1.0 - 7034
- Monitor(s): Samsung CRG9
- Apple Silicon/M1 (yes or no): yes
```

**Comment by waydabber** [comment]
I am not sure I fully understand th eproblem. MonitorControl uses a combination of software and hardware dimming - if hardware brightness (backlight) is set to zero, additional software dimming occurs which does not affect the display's OSD brightness level. Let me know if I misunderstand the issue.

I'll convert this to a discussion.
